docs/reactive-resume_installation.md

```diff
@@ -0,0 +1,37 @@
+## Reactive Resume installation
+This image using reactive-resume last release and PR https://github.com/AmruthPillai/Reactive-Resume/pull/906 by https://github.com/dvd741-a that enable to store pictures at local disk instead S3 storage.
+
```

Comment:
Can you add a comment about running the pre-install script first?

I don´t know if there is any other information required, like showing a print from the app and which app from the template should be selected, just for completeness.

Reply:
I removed the info as it already merged into latest release, and just add a point to run `install_reactive-resume.sh` before continuing installation